File: ios/Classes/message/entity/group_tips/GroupTipsElemMemberInfoEntity.swift
import ImSDK;

//  Created by 蒋具宏 on 2020/2/15.
// 群通知节点成员信息实体
public class GroupTipsElemMemberInfoEntity : NSObject{
    var identifier : String?;
    var shutupTime : UInt32?;
    
    required public override init() {
     }
    
    init(info : TIMGroupTipsElemMemberInfo) {
        super.init();
        self.identifier = info.identifier;
        self.shutupTime = info.shutupTime;
    }
}

File: lib/entity/group_tips_elem_member_info_entity.dart
/// 群tips，成员变更信息
class GroupTipsElemMemberInfoEntity {
  /// 消息内容
  String identifier;

  /// 被禁言时间
  int shutupTime;

  GroupTipsElemMemberInfoEntity.fromJson(Map<String, dynamic> json) {
    identifier = json["identifier"];
    shutupTime = json["shutupTime"];
  }
}
